packages/styles/src/components/header/index.scss

```diff
@@ -140,6 +140,14 @@ post-header {
     }
   }
 
+  a:not(post-mainnavigation *, post-language-switch *) {
+    display: flex;
+
+    @include media.max(sm) {
+      padding: 4px 10px;
```

Comment:
I guess this padding should be applied to buttons as well, you can probably include it to this mixin: https://github.com/swisspost/design-system/blob/main/packages/styles/src/components/header/_mixins.scss#L3